db/migrate/20200207104825_create_books.rb

```diff
@@ -0,0 +1,15 @@
+class CreateBooks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
```

Comment:
Just a note that we will need to figure out what kind of DB indexes it'd be good to add in this table. Indexes will increase the performance of certain SQL queries and will be useful when we add the search functionality. Don't have to do it here though, they can always be added with a separate migration at a later stage.

Reply:
Thanks for this comments. I will keep it in mind.